charts/consul/templates/crd-gatewaypolicy.yaml

```diff
@@ -0,0 +1,210 @@
+{{- if .Values.connectInject.enabled }}
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
did the go structs already make it in or did you hand roll these?

Reply:
just wondering b/c normally these are generated from go structs

Reply:
Take a look at @sarahalsmiller's PR for an example: https://github.com/hashicorp/consul-k8s/pull/2735/files#diff-ecdac93bf4d514eb648c162d8b54671660e679f0b730829a66ad944e70ca7540

Reply:
I hand rolled them, talked to @sarahalsmiller on slack and realized that I should be generating them from structs, going to close this and re-open it with the correct setup